modules/021-kube-proxy/templates/daemonset.yaml

```diff
@@ -69,7 +69,11 @@ spec:
           name: config
         - mountPath: /var/lib/kube-proxy-cm
           name: kube-proxy-cm
+        resources:
+          requests:
+            {{- include "helm_lib_module_ephemeral_storage_logs_with_extra" 10 | nindent 12 }}
       - name: nodeport-bind-address
+        {{- include "helm_lib_module_container_security_context_read_only_root_filesystem_capabilities_drop_all" . | nindent 8 }}
```

Comment:
list all components that will be restarted in impact description.

Reply:
done